examples/nodejs-ts-app-multi-entrypoint/README.md

```diff
@@ -0,0 +1,54 @@
+# Multi-entrypoint OPA-WASM node demo script
```

Comment:
👏 Thank you!

At some point, I guess we can cut down the number of example apps we cultivate, since this one subsumes one of the others, via its "default entrypoint case". But we can do that gardening in a follow-up 🌻